versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/tutorials/deploying-the-iperf-application.mdx

```diff
@@ -70,18 +73,22 @@ The following information is required.
 | <slice configuration\> | The name of the slice configuration file |
 | <project namespace\> | The project namespace on which you apply the slice configuration file. |
 
-Switch the context to the KubeSlice Controller using the following command:
-```
-kubectx <cluster name>
-```
+Perform these steps:
 
-Run the following command to apply the manifest file on the project namespace:
-```
-kubectl apply -f <slice configuration>.yaml -n <project namespace>
-```
+1. Switch the context to the KubeSlice Controller using the following command:
+    ```
+    kubectx <cluster name>
+    ```
+
+2. Apply the manifest file on the project namespace using the following command:
+    ```
+    kubectl apply -f <slice configuration>.yaml -n <project namespace>
+    ```
 
 ## Deploying iPerf
-In this tutorial, iperf-sleep and iperf-server will be deployed in the two different clusters. The cluster used for iperf-sleep is referred to as <sleep cluster\>, and the cluster used for iperf-server is referred to as <server cluster\>.
+In this tutorial, iperf-sleep and iperf-server will be deployed in the two different clusters. 
+The cluster used for iperf-sleep is referred to as <sleep cluster\>, and the cluster used for iperf-server is 
```

Comment:
why do we need a back slash - please remove

Reply:
![image](https://user-images.githubusercontent.com/105046156/180399694-71255b23-878c-4983-8fa6-abd85066f2f4.png)

If the back slash is removed, then the MD throws the error. I have attached the screenshot for your reference.

Reply:
check for <cluster name> i have removed the \ and compile throws the error.